packages/core/src/context/internal/AgenticaOperationComposer.spec.ts

```diff
@@ -0,0 +1,314 @@
+import type { IHttpLlmFunction, ILlmFunction, IValidation } from "@samchon/openapi";
+
+import { describe, expect, it } from "vitest";
+
+import type { IAgenticaConfig } from "../../structures/IAgenticaConfig";
+import type { IAgenticaController } from "../../structures/IAgenticaController";
+import type { IMcpLlmFunction } from "../../structures/IMcpLlmFunction";
+
+import { compose, divide, getOperations, toClassOperations, toHttpOperations, toMcpOperations } from "./AgenticaOperationComposer";
+
+// 테스트 헬퍼 함수들
```

Comment:
Korean.

Reply:
yeah it's so sad, it's target for to fix....